robots.cc

```diff
@@ -66,8 +66,9 @@ class RobotsMatchStrategy {
   static bool Matches(absl::string_view path, absl::string_view pattern);
 };
 
-// Returns true iff URI path matches the specified pattern. Pattern is anchored
-// at the beginning of path. '$' is special only at the end of pattern.
+// Returns true if and only if URI path matches the specified pattern. Pattern
```

Comment:
We meant a regular 'if' here. I am adding suggestions to make it look like before, but instead of "iff", it should say "if" (hopefully, I did the suggestions correctly)

```suggestion
// Returns true if URI path matches the specified pattern. Pattern is anchored
```